website/docs/components/page-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,237 @@
+## Usage
+
+### When to use
+
+- To display the title or topmost heading within a page.
+- To communicate metadata pertaining to the page; e.g., status, description, subtitle, etc.
+- To highlight page-level actions and functions.
+
+### When not to use
+
+- To communicate page-level information anywhere other than the top of the page.
+
+## Title
+
+Displays the title or topmost heading of the page. This should be limited to no more than four words and not exceed 50 characters.
+
+### Multiple `<h1>` elements
+
+Since the Page Header renders the title in an `<h1>` element, ensure that this is the only `<h1>` on the page. More than one `<h1>` can confuse screen readers and impact the semantic hierarchy of the page.
+
+## Size
+
+Size only pertains to the Figma component and accounts for smaller viewports by stacking elements. The Ember component supports a basic level of fluidity and responsiveness out of the box.
+
+Refer to [responsiveness](#responsiveness) for more information on how the component adapts to different viewport and container sizes.
+
+### Large
+
+![Large Page Header variant](/assets/components/page-header/page-header-size-large.png =945x*)
+
+### Small
+
+![Small Page Header variant](/assets/components/page-header/page-header-size-small.png =300x*)
+
+## Breadcrumb
+
+Displays a Helios [Breadcrumb](/components/breadcrumb) to communicate the application hierarchy and location to the user.
+
+![Page Header Breadcrumb](/assets/components/page-header/page-header-breadcrumb.png =700x*)
```

Comment:
Suggest to update this example to match with the third screenshot in this ticket: https://hashicorp.atlassian.net/browse/HCPF-639

Reasons: 
- The first item for Organization uses an "org" icon. Suggested to be updated to the "dashboard" icon to match better with the type of the page (an org dashboard).
- The second item uses the outdated icon choice for "project" .  It has been updated to use the "doc" icon because we want to reserve the "folder" icon for the future resource hierarchy called "Folder", which sits in between Organization and Projects. 
- With multi-project, we introduced a project listing page which is currently missing in the breadcrumb

Reply:
Great, thanks for the notes on outdated icons, I'll update! There will likely be other screens and examples that I'll have to clean up as well.